test/bases/renku_data_services/data_api/test_projects.py

```diff
@@ -1243,7 +1243,7 @@ async def test_project_copy_basics(
     assert copy_project == snapshot(exclude=props("id", "updated_at", "creation_date", "etag", "template_id"))
 
     _, response = await sanic_client.get(
-        f"/api/data/projects/{copy_project["id"]}", params={"with_documentation": True}, headers=user_headers
+        f"/api/data/projects/{copy_project['id']}", params={"with_documentation": True}, headers=user_headers
```

Comment:
I think your formatter is set up weird, using different quotes in this circumstance isn't necessary anymore in newer python (not that it matters, though)

Reply:
Yeah this is a problem. And I dont want to get into writing a complicated migration right now. Because I cannot just set all slugs to lowercase in the migration. I have to also handle potential conflicts and rename some slugs so that when I make all lowercase I do not get conflicts with duplicates in the DB. So I will leave this for a later PR.

I will close this and reopen another PR where I just change the validation when dumping namespaces. We can do this without validation when we respond.